tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -40,6 +40,7 @@ namespace Datadog.Trace
     internal class TracerManagerFactory
     {
         private const string UnknownServiceName = "UnknownService";
+        private const ulong DefaultTimeoutMs = 100_000;
```

Comment:
nit: I think we should inline this, as it's too general a name, and only used in one place. If/when we unify this across the managed code too, we can extract it out where it _should_ be (which probably isn't here anyway 😄)

```suggestion
```